helm/fullnode/templates/tests/test-fullnode-connection.yaml

```diff
@@ -14,5 +14,18 @@ spec:
         - -c
         - |-
           set -ex
-          curl --fail -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0", "method":"get_metadata","params":[],"id":1}' http://{{ include "diem-fullnode.fullname" . }}-lb:80
```

Comment:
This test was probably never synced back. We should stick with the `get_metadata` method